staging/src/k8s.io/apiserver/pkg/admission/plugin/validatingadmissionpolicy/controller_reconcile.go

```diff
@@ -48,7 +50,11 @@ type policyController struct {
 
 	// Provided to the policy's Compile function as an injected dependency to
 	// assist with compiling its expressions to CEL
-	ValidatorCompiler
+	cel.FilterCompiler
+
+	Matcher
```

Comment:
If I'm understanding this right, anonymous fields here will export the fields do we want that? All the other fields are unexported.

Reply:
Right I'll make it private